ci-cd/aws-az-cli-chamber-helm/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM segment/chamber:2.10.6 AS chamber
+# FROM mcr.microsoft.com/azure-cli:2.9.1 AS azure-cli
```

Comment:
I think this can go / needs to come back?

Reply:
Yes this was just for testing purposes which didnt work out in the end.